client/src/components/overview/overview.jsx

```diff
@@ -1,33 +1,121 @@
+      },
+      // sizeAmount State
+      chosenSize: '-',
+      chosenAmount: '',
+      skuID: '',
+      availableSizes: [{ size: '-', quantity: '8' }],
+      availableAmount: '-',
+      // Photos
+      mainPhoto: '',
+      thumbnails: [{
+        thumbnail_url: 'https://images.unsplash.com/photo-1501088430049-71c79fa3283e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=300&q=80',
+      }],
+      showWarning: false,
+      showModal: false,
+      currentStyle: '',
+      thumbnailIndex: '0',
+
+      original_price: '140.00',
+      sale_price: '0',
+
     };
+    this.setSelectedStyle = this.setSelectedStyle.bind(this)
+    this.setCurrentSku = this.setCurrentSku.bind(this)
+    this.setCurrentStyleInfo = this.setCurrentStyleInfo.bind(this)
+    this.setChosenSize = this.setChosenSize.bind(this)
+    this.setAvailableAmount = this.setAvailableAmount.bind(this)
+    this.setChosenAmount = this.setChosenAmount.bind(this)
+    this.setMainPhoto = this.setMainPhoto.bind(this)
+    this.setThumbnails = this.setThumbnails.bind(this)
+    this.setAvailableSizes = this.setAvailableSizes.bind(this)
+    this.setSkuID = this.setSkuID.bind(this)
+    this.setShowWarning = this.setShowWarning.bind(this)
+    this.setShowModal = this.setShowModal.bind(this)
+    this.setThumbnailIndex = this.setThumbnailIndex.bind(this)
+    this.setOriginalPrice = this.setOriginalPrice.bind(this)
+    this.setSalePrice = this.setSalePrice.bind(this)
   }
+  setSelectedStyle(selectedStyle) { this.setState({ selectedStyle }) }
+  setCurrentSku(currentSku) { this.setState({ currentSku }) }
+  setCurrentStyleInfo(currentStyleInfo) { this.setState({ currentStyleInfo }) }
+  setSkuID(skuID) { this.setState({ skuID }) }
 
-  componentDidMount() {
-    this.setState({
-      info: this.props.overview,
-      styles: this.props.styles
-    })
+  setChosenSize(chosenSize) { this.setState({ chosenSize }) }
+  setChosenAmount(chosenAmount) { this.setState({ chosenAmount }) }
+  setAvailableSizes(availableSizes) { this.setState({ availableSizes }) }
+  setAvailableAmount(availableAmount) { this.setState({ availableAmount }) }
+
+  setMainPhoto(mainPhoto) { this.setState({ mainPhoto }) }
+  setThumbnails(thumbnails) { this.setState({ thumbnails }) }
+  setShowWarning(showWarning) { this.setState({ showWarning }) }
+
+  setShowModal(showModal) { this.setState({ showModal }) }
+  setThumbnailIndex(thumbnailIndex) { this.setState({ thumbnailIndex }) }
+  setOriginalPrice(original_price) { this.setState({ original_price }) }
+  setSalePrice(sale_price) { this.setState({ sale_price}) }
+
+  jest() {
+    return 'Jest'
   }
 
-  render() {
-    let renderFeatures, renderStyles
-    if (this.state.info !== '') {
-      renderFeatures = <Features features={this.state.info.features} />
-      renderStyles = <Styles styles={this.state.styles.results} />
+
+  componentDidMount() {
+    // console.log('bigProps', this.props)
+    if (this.props === undefined) {
```

Comment:
What happens if this.props === undefined? the block is blank.

Reply:
Nothing happens, it used to have something there, but I deleted it and didn't refactor so it looks just empty.